### PR TITLE
Upgrade crossplane version to 1.10.0

### PR DESCRIPTION
**Pull request by susana-garcia**
## Checklist

- [X] PR contains a single logical change (to build a better changelog).
- [ ] Update the documentation.
- [X] Categorize the PR by setting a good title and adding one of the labels:
      `bug`, `enhancement`, `documentation`, `change`, `breaking`, `dependency`
      as they show up in the changelog.

### CODE REVIEW

File: tests/golden/defaults-with-provider/crossplane/crossplane/01_helmchart/crossplane/templates/clusterrole.yaml [hunk 1]
```diff
@@ -12,8 +12,8 @@ metadata:
     app.kubernetes.io/managed-by: Helm
     app.kubernetes.io/name: crossplane
     app.kubernetes.io/part-of: crossplane
-    app.kubernetes.io/version: 1.9.0
-    helm.sh/chart: crossplane-1.9.0
+    app.kubernetes.io/version: 1.10.0
```
Comment (zugao) [inline tests/golden/defaults-with-provider/crossplane/crossplane/01_helmchart/crossplane/templates/clusterrole.yaml:15]: Any reason why we haven't gone directly with version 1.11?

**Review by zugao — APPROVED** [review]
I would release a new version in lab cluster only, thus skipping the default repo update. I would leave it for some time to see if there are no issues whatsoever.

File: tests/golden/defaults-with-provider/crossplane/crossplane/01_helmchart/crossplane/templates/clusterrole.yaml (hunk 1 above)
Comment (susana-garcia) [reply tests/golden/defaults-with-provider/crossplane/crossplane/01_helmchart/crossplane/templates/clusterrole.yaml:15]: as far as I know, the latest stable is 1.10

Comment (zugao) [reply tests/golden/defaults-with-provider/crossplane/crossplane/01_helmchart/crossplane/templates/clusterrole.yaml:15]: Yes you are right, though there's a helm chart 1.11 already which has a `rc` on crossplane 1.11. It's a bit confusing

**Comment by susana-garcia** [comment]
> I would release a new version in lab cluster only, thus skipping the default repo update. I would leave it for some time to see if there are no issues whatsoever.

I'm not sure what you mean, sorry. Do you mean that I should create a new tag and update crossplane version with it in the lab cluster?

**Comment by zugao** [comment]
> I'm not sure what you mean, sorry. Do you mean that I should create a new tag and update crossplane version with it in the lab cluster?

don't release anything just yet as renovate will update the defaults to the latest version. Just merge to master then deploy in the lab cluster.

**Pull request merged**